### PR TITLE
[postgres] Move `scanTableQuery` to `scan.go`

### PR DESCRIPTION
**Pull request by nathan-artie**

### CODE REVIEW

File: lib/postgres/queries.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package postgres
+
+import (
+	"github.com/jackc/pgx/v5"
+)
+
+func QuotedIdentifiers(ids []string) []string {
```
Comment (nathan-artie) [inline lib/postgres/queries.go:7]: Just moved this from `queries/queries.go`, no changes.

File: lib/postgres/scan.go [hunk 2]
```diff
@@ -45,21 +46,65 @@ func (t *Table) NewScanner(db *sql.DB, batchSize uint, errorRetries int) scanner
...
+	StartingKeys []string
+	// Second where clause
+	SecondWhere Comparison
+	EndingKeys  []string
+
+	OrderBy []string
+	Limit   uint
+}
+
+func scanTableQuery(args scanTableQueryArgs) string {
```
Comment (nathan-artie) [inline lib/postgres/scan.go:80]: No changes other than `quotedIdentifiers` -> `QuotedIdentifiers`

File: lib/postgres/scan.go [hunk 3]
```diff
@@ -45,21 +46,65 @@ func (t *Table) NewScanner(db *sql.DB, batchSize uint, errorRetries int) scanner
 	}
 }
 
+type Comparison string
```
Comment (nathan-artie) [inline lib/postgres/scan.go:49]: No changes to this block other than `quotedIdentifiers` -> `QuotedIdentifiers`

File: lib/postgres/queries_test.go [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package postgres
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestQuotedIdentifiers(t *testing.T) {
```
Comment (nathan-artie) [inline lib/postgres/queries_test.go:9]: No changes.

**Review by Tang8330 — APPROVED** [review]

**Pull request merged**